pkgs/development/tools/misc/autogen/default.nix

```
@@ -79,7 +79,7 @@ stdenv.mkDerivation rec {
   '' + stdenv.lib.optionalString (!stdenv.hostPlatform.isDarwin) ''
     # remove /build/** from RPATHs
```

```suggestion
    # remove build directory (/build/**, or /tmp/nix-build-**) from RPATHs
```
might make it a little more clear why this change was needed